x-pack/plugins/security_solution/public/cases/containers/configure/use_connectors.tsx

```diff
@@ -31,7 +31,9 @@ export const useConnectors = (): ReturnConnectors => {
         const res = await fetchConnectors({ signal: abortCtrl.signal });
         if (!didCancel) {
           setLoading(false);
-          setConnectors(res);
+          setConnectors(
+            res.filter((connector) => !connector.consumer || connector.consumer === 'case')
```

Comment:
I think it is better to filter the connectors at the backend (cases server)

Reply:
Replaced API consumer for actions with a ServiceNow specific config setting isCaseOwned